src/components/designer/taro/actions/MintAssetModal.spec.tsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/dom';
+import { waitFor } from '@testing-library/react';
+import { Status } from 'shared/types';
+import { BitcoindLibrary } from 'types';
+
```

Comment:
nit: remove extra newlines.

If you use VS Code, there is an automatic [import sorter plugin](https://marketplace.visualstudio.com/items?itemName=mike-co.import-sorter) which manages these based on rules defined in [settings.json](https://github.com/amovfx/polar/blob/0ad7f8b695aefe05e3096a8ebe31004548764751/.vscode/settings.json#L20)